.husky/commit-msg

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+REGEX_COMMIT_MSG="^((NUT)-[0-9]{1,15}: .+|Merge branch .+|Merge remote-tracking branch .+|Revert .+)$"
+
+if ! grep -iE "$REGEX_COMMIT_MSG" "$1"; then
+  echo -e \
+    '\e[31merror\e[0m commit messages must begin with a JIRA identifier' \
+    '(e.g. NUT-100: Did the thing)'
```

Comment:
we probably want to discuss this - this is technically open-source and people who contribute may not have a jira reference to relate back to

Reply:
shall i remove this step then ?

Reply:
Yes I think so